components/console.tsx

```diff
@@ -10,12 +10,41 @@ import {
 } from 'react';
 import { ConsoleOutput } from './block';
 import { cn } from '@/lib/utils';
+import Image from 'next/image';
 
 interface ConsoleProps {
   consoleOutputs: Array<ConsoleOutput>;
   setConsoleOutputs: Dispatch<SetStateAction<Array<ConsoleOutput>>>;
 }
 
+// Helper function to detect base64 images
+function isBase64Image(str: string): boolean {
```

Comment:
This function isn't being used

Reply:
Fixed in the following commit: [https://github.com/vercel/ai-chatbot/pull/649/commits/df25edf2519d70dfe28238438b3440f28dc21e76](https://github.com/vercel/ai-chatbot/pull/649/commits/df25edf2519d70dfe28238438b3440f28dc21e76)